### ISSUE TITLE
Bulk Edit

### ISSUE THREAD

**Issue by ivandortulov** [OpenShot/openshot-qt#5224]
**Describe the new feature:**
I would like to be able to set properties for multiple videos at once.

**Describe the solution you'd like:**
By selecting multiple clips in the Timeline (using CTRL), the Property panel shows common properties. I would like to edit those properties and have the values applied to all videos. At the start of the video.

Another alternative would be to have copied clips be linked with their originals. Then when editing the original, you edit the linked copy. And there should be a button to unlink (Make unique) so you could still retain the original functionallity.

**Describe alternatives you have considered:**
I just go and edit one clip and then copy paste it. However, if I now make changes to the original clip, I have to go around and make the same changes to the other clips.

**Screenshots:** (Optional)
![ezgif-2-b98315851e](https://user-images.githubusercontent.com/28752782/234207434-50349dd3-e6f8-48e7-9f6f-00c2c0adcdb8.gif)

**Comment by Colorjet3** [comment]
Hello @ivandortulov There is a limited functionality of this type but limited to Alpha, Scale, Rotation, Time, and Volume.

1.  Drag 3 emojis onto Track 5.  You can have them snapped together or however you want them on the timeline.  They can be on different tracks as well.
2.  Now select the first emoji and from the Properties window modify the Scale X & Y and Location X & Y.
3.  While the first emoji is selected, right click on it, select Copy | Key Frames | All.
4.  Now select the other two emojis leveraging the CTRL + Mouse select.  Once you have the clips selected, right click and Paste.

May be additional attributes need to be made available as well.

**Comment by ivandortulov** [comment]
@Colorjet3 Thank you for that. I didn't know I could do it that way. For the time being, I will make use of this suggestion. However, I do think that a bulk edit similar to what I proposed would provide a better user experience by saving time for the user.

**Comment by Colorjet3** [comment]
Hello @ivandortulov You are welcome.  I already submitted and "Enhancement" request to add the "Shear" attribute to this list as well as I am sure it was an oversight when the "Shear" feature was added.

The rest of your suggestion is going to take some thought process and planning to implement, if approved by the lead developer.  Not everyone is going to like the fact that "copied" clips are linked and changes to the original ultimately will change other linked clips.  Obviously, the solution would be the you choose to link objects or not.  This is all achievable but does add complexity and will take time to architect, test, and make available.

I am going to close this ticket for now but feel free to add other ideas and thoughts and I can add those to an internal ticket that I will be creating so we don't lose track of this.

Best of luck with your projects.
